vsphere/datadog_checks/vsphere/utils.py

```diff
@@ -67,19 +74,20 @@ def is_resource_excluded_by_filters(mor, infrastructure_data, resource_filters):
             return False
 
     if hostname_filter and isinstance(mor, vim.VirtualMachine):
-        host = infrastructure_data.get(mor).get("runtime.host")
-        hostname = infrastructure_data.get(host, {}).get("name", "")
+        host = infrastructure_data[mor].get("runtime.host")
+        hostname = infrastructure_data.get(host, {}).get("name", "")  # type: ignore
```

Comment:
Actually, the `type ignore` is not needed, removing it.

Reply:
This will lead to another error:

```
datadog_checks/vsphere/utils.py:79:64: error: Invalid index type "Type[ManagedEntity]" for "Dict[ManagedEntity, InfrastructureDataItem]"; expected type "ManagedEntity"
```

Reply:
Fixed.